integration-framework/src/main/java/org/apache/cassandra/sidecar/lifecycle/InJvmDTestLifecycleProvider.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.lifecycle;
+
+import java.util.concurrent.TimeUnit;
+
+import org.apache.cassandra.distributed.api.IInstance;
+import org.apache.cassandra.sidecar.cluster.instance.InstanceMetadata;
+
+/**
+ * Manages the lifecycle of JVM Dtest Casasndra instances.
+ * This should used for integration tests where Cassandra instances are started and stopped
+ */
+public class InJvmDTestLifecycleProvider implements LifecycleProvider
+{
+    private final Iterable<? extends IInstance> instances;
+
+    public InJvmDTestLifecycleProvider(Iterable<? extends IInstance> instances)
+    {
+        this.instances = instances;
+    }
+
+    public void start(InstanceMetadata instanceMetadata)
```

Comment:
let's add override annotations in this class
```suggestion
    @Override
    public void start(InstanceMetadata instanceMetadata)
```